project/Build.scala

```diff
@@ -138,6 +138,9 @@ object Build {
   // Run tests with filter through vulpix test suite
   val testCompilation = inputKey[Unit]("runs integration test with the supplied filter")
 
+  // Run code coverage instrumentation tests
+  val testCoverage = inputKey[Unit]("runs code coverage instrumentation test")
```

Comment:
I'm a bit wary of adding more ways to run tests, could we instead have testCompilation run the coverage tests too?

Reply:
sure!